setup.py

```diff
@@ -495,6 +495,7 @@ def package_tree(pkgroot):
 
 package_path(join(SERVER, 'static'))
 package_path(join(ROOT, 'bokeh', 'core', '_templates'))
+package_path(join(ROOT, 'bokeh', 'server', 'views'), ('.html'))
```

Comment:
I think it's fine where it is. Since it's a template it itself is never served directly like things in static are. 

> On Sep 2, 2016, at 06:36, Tim Snyder <notifications@github.com> wrote:
> 
> In setup.py:
> 
> > @@ -495,6 +495,7 @@ def package_tree(pkgroot):
> >  
> >  package_path(join(SERVER, 'static'))
> >  package_path(join(ROOT, 'bokeh', 'core', '_templates'))
> > +package_path(join(ROOT, 'bokeh', 'server', 'views'), ('.html'))
> @bryevdv looking at this patch again after some sleep, is it a better fix to move bokeh/server/views/app_index.html into bokeh/server/static/html? It would also require a oneline change to the server code. I wouldn't mind redoing this if you think that's better.
> 
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub, or mute the thread.
>